packages/opentelemetry-scope-zone/README.md

```diff
@@ -0,0 +1,69 @@
+# OpenTelemetry Scope Zone
+[![Gitter chat][gitter-image]][gitter-url]
+[![NPM Published Version][npm-img]][npm-url]
+[![dependencies][dependencies-image]][dependencies-url]
+[![devDependencies][devDependencies-image]][devDependencies-url]
+[![Apache License][license-image]][license-image]
+
+This module provides *Zone Scope Manager with bundled [zone-js]* for Web applications.
+If you have your own [zone-js] please use *@opentelemetry/scope-zone-peer-dep*
```

Comment:
Should we mention that if you are using Angular your app likely already has Zone.js?

Reply:
Could you please add link for `@opentelemetry/scope-zone-peer-dep`.